### PR TITLE
TASK: New NodeTypes and subtle changes for existing NodeTypes

### PR DESCRIPTION
**Pull request by Inkdpixels**
* Fixed some unwanted responsive behaviors
* Adjusted some constraints
* Re-Added the alignment properties to the Image and ImageWithText NodeTypes
* Added the ruler NodeType
* Fixed a font-rendering issue within the keyvisual

### CODE REVIEW

**Comment by dfeyer** [comment]
Just a small comment about the Ruler node type (check https://neos-project.slack.com/archives/guild-website/p1459238178000145)

For me this kind of separator must not be part of the editor responsibility. But it depend on content. By ex. i prefer to have some kind of « Section » node type (that output a valid section html element (and render a simple content collection), and the section display the border bottom or what ever is needed as separator. It’s an important split between content and layout. If during a next relaunch we decide to not use the HR anymore … we have to hack content, if we have proper Section node type we can just style the content differently. It’s also useful to have more semantic markup

**Comment by Inkdpixels** [comment]
@dfeyer Thanks for the kind feedback, really appreciated! :) Will tackle this as soon as possible! :+1:

File: Packages/Sites/Neos.NeosIo/Configuration/NodeTypes.Stage.yaml [hunk 1]
```diff
@@ -20,6 +20,7 @@
           'TYPO3.Neos.NodeTypes:TextWithImage': TRUE
           'TYPO3.Neos.NodeTypes:Headline': TRUE
           'Neos.NeosIo:Button': TRUE
+          'Neos.NeosIo:ChoiceButtons': TRUE
```
Comment (Sebobo) [inline Packages/Sites/Neos.NeosIo/Configuration/NodeTypes.Stage.yaml]: Code style : We discussed in our call with the other website people to use lowercase true/false in yaml as standard

File: Packages/Sites/Neos.NeosIo/package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "build:scripts": "browserify -t [ babelify ] -p browserify-derequire Resources/Private/JavaScript/index.js > Resources/Private/JavaScript/Bundle.min.js",
     "fix:amd": "fandr Resources/Public/JavaScript/Bundle.min.js '{\"&& define.amd\": \"&& false\"}'",
     "build:sass": "node-sass Resources/Private/Scss/ -o Resources/Public/Styles/",
-    "postbuild:sass": "postcss --use autoprefixer Resources/Public/Styles/*.css -d Resources/Public/Styles/",
```
Comment (Sebobo) [inline Packages/Sites/Neos.NeosIo/package.json]: Personally I think having tasks in the package file is great, but it's really hard to read and maintain with the longer commands.
Grunt/Gulp deliver a bit more clarity there in my opinion. But we can discuss this :)

**Comment by Sebobo** [comment]
Hey @Inkdpixels looks great, I left a few comments in their respective commits.

**Pull request merged**